apps/store/themes/store/js/list_assets.js

```diff
@@ -490,114 +490,116 @@ function resolveDomain() {
 }
 
 $(window).load(function () {
+    if (store.taxonomyAvailability) {
```

Comment:
Thank you Sameera . This entire code block has been refactored with multiple taxonomy. Its not available anymore.